zomes/blog/entry_types/post/validation.rs

```diff
@@ -0,0 +1,23 @@
+extern crate hdk;
+#[macro_use]
+extern crate serde_derive;
+
+struct Post {
+    content: String,
+    date_created: String
+}
```

Comment:
This struct should be defined outside this file and then included here an in other WASM modules.